packages/loot-core/src/client/shared-listeners.ts

```diff
@@ -47,40 +49,46 @@ export function listenForSyncEvent(actions, store) {
       }
     } else if (type === 'error') {
       let notif: Notification | null = null;
-      const learnMore =
-        '[Learn more](https://actualbudget.org/docs/getting-started/sync/#debugging-sync-issues)';
+      const learnMore = i18n.t(
```

Comment:
same question here and other places

Reply:
in `loot-core` I use the i18next package directly which depends on a global singleton. since we don't have React components here we aren't able to use the hook. I think a slightly different convention in a different package is okay, what do you think?

Reply:
Good point @jfdoming . Ideally we should use the same approach everywhere. So we could do `import { t } from 'i18next';` in loot-core and use that here too.